language/moe/README.md

```diff
@@ -0,0 +1,230 @@
+# Reference Implementation for Mixtral-8x7B-v0.1
+
+**Basic implementation for Mixtral-8x7B-v0.1. Few noteworthy items:**
+
++ Processing of Validation dataset is not finalized yet. Decision on input token lengths is pending
```

Comment:
It's 5k samples which are sampled randomly from 3 datasets: open_orca_gpt4, GSM8k and MBXP